src/hidimstat/utils.py

```diff
@@ -24,13 +24,16 @@ def quantile_aggregation(pvals, gamma=0.5, gamma_min=0.05, adaptive=False):
 
 def fdr_threshold(pvals, fdr=0.1, method="bhq", reshaping_function=None):
     if method == "bhq":
-        return _bhq_threshold(pvals, fdr=fdr)
+        threshold = _bhq_threshold(pvals, fdr=fdr)
```

Comment:
I don't see the benefit of that change. (But I don't really mind, let's say it's -0)

Reply:
It's to have improved the self-explanatory of the code, without using the implicit of the output of the function.
And it will help for adding assertions in the future.